.travis.yml

```diff
@@ -5,17 +5,22 @@ services:
 
 env:
   global:
+    # TODO convert to DOCKER_HUB_USERNAME and DOCKER_HUB_PASSWORD for
+    # Numenta's docker hub account
     # QUAY_USERNAME and QUAY_PASSWORD for docker image upload
-    - secure: "lKaTzEL6UNiEfp+BWLOUILG9BMtjwEMpwt6Yag0cQGHix7qJ/ElZ0t3oFw6ZwuDmA5qceAXIdxHLUK9HGVI2MloLk8czGhjvtfJ4XhOxtEJRQ0VkDGPsKN4cfhB4ZjGo6GAPtNqStMyNiY7BZuTrZa7coDLCoUeYcOmTpi6pmd1rrkk725B9QCTuhFHbPhkuL2yu/Jk6WxkHJBKjmuZek+iQa7lRItgMrG0/319PXLvwIGGl00nLFy+Ly5Ciwzux4wuHLTySZQKu0H9FX81A7smM0FW/42kg3ckGa2qLxRw/Pi8Nm/aIk8LD0QXzI5N7HhFfidOTgDS8Mt1HgfxmTk4wUXZ/KvCCshqjimzMc/s9i9wPZX9UqqcfrpZkmwz8dzhm1bndN45ZOCy6xAYT6dzf8T4mLMDjVWSW4+DUoW4sYHRLVujjcMk7ybcwGV43VruPTJnc8XVAhT+VIMQkoPjhQmTOn8h82LRNGYtLa5RReCh9OPKVYB2Quz18FXMWgFt7A6VWudL0c7/8CusLvuo+pLcxt9pnV40rvu1YEohpEj8qR/qTSaDUBZM0J9SVf5zrZR80pZUnXkDF8nm+mcLOTley3YWipU19lCR7dzVyCAiQdVAuNPdnyem3Yk8enGkAJbfLd6eaIDs+p73D0JXh1Nx1px1movVLQH3ohIw="
-    - secure: "w1614pomHLltkBhqWM2bOvbymFWIWKqSqqIBDvaNn9tbQScioItJoELBT7g7+cD7nyU7OvpQ1U2fk0xVkCeNvYU0xS1vP4o/VnZRpup7f7Tkiq+2rf4fjwYr3HHnJjwak1l9bsw6FkgzKaVvSdiUJHMVxiIuLd3fVozR7qjBBhTDxSlWGOpSgd+ttpgMZwU5zQjdaVQr1D7E8M0979ZnWMrNRyLiAUeHaPILS815b+ijgqR+i5nmu0/FTCGM9Ik4KIzIfWq8AdfPdbRiq8c+LrrTPfyKcIQJaHmfduYRM4LycGWwzkXFBNtLrJ7uFLG9RDVemOHuHOWIJX8qCUIV4XuESXxH3fUQr6r+yxquTJbzXxNtoaLa6tBOTQWKDrRjT4z9Mf9Im14F2V59EUDoQowHx5bjunOH5wg3ruYNKYYBFRYra5kx0CkKrqFBzyl8fTUEQLyx1HWTVUC1WTXEeD/aFKOSIxW5DxZr5W4LLlW2+Raa52ZzY28Q6AdueFQCRzoJ70/GsJRlSsBdWNOHN4gSp1cZuToLWY15y64QhAMVDpikB+V4hmkbceLiTqeWzTStNL1sa32RHr6i/9zeFZw1pMD1+eOg9x6fgODfh2sqr/zPbu2oONsHnc4D2jwsEax4o+Dv5QHLvK7jdyWUmu47a9QReoexXK60jZXs3CA="
+    - secure: ""
+    - secure: ""
 
 script:
-  - docker build --rm -t quay.io/pypa/manylinux1_i686:$COMMIT -f docker/Dockerfile-i686 docker/
-  - docker build --rm -t quay.io/pypa/manylinux1_x86_64:$COMMIT -f docker/Dockerfile-x86_64 docker/
+  # We're only interested in x86_64.
+  - docker build --rm -t numenta/manylinux1_x86_64_centos6:$COMMIT -f docker/Dockerfile-x86_64 docker/
 
 
 deploy:
   provider: script
+  # TODO Update docker/deploy.sh to work with numenta's environment. Better
+  # yet, make it an autobuild in docker hub repo
```

Comment:
The decision to go with quay.io was the result of an investigation by @BoltzmannBrain.  Unfortunately, I don't have the JIRA available, but Docker Hub, Quay.io, self-hosted, an AWS, among others were considered.  If I recall correctly, one of the stand-out reasons was better team-management features.

Regardless, I bring it up not because I feel quay.io is better (that's beside the point), but because it was decided as a group to use quay.io, and it's part of the documentation at https://numenta-internaldocumentation.pbworks.com/w/page/108045989/Bamboo%20CI. i.e. the informed decision was made already, and we should stick to it.